cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -244,8 +252,10 @@ def test_tag(provisioner, prov_data, provider, vm_name):
     vm = provisioner(template_name, prov_data)
 
     tags = vm.get_tags()
-    assert any(tag.category.display_name == "Service Level" and tag.display_name == "Gold"
-               for tag in tags), "Service Level: Gold not in tags ({})".format(str(tags))
+    assert any(
+        tag.category.display_name == "Service Level" and tag.display_name == "Gold"
+        for tag in tags
+    ), "Service Level: Gold not in tags ({})".format(str(tags))
```

Comment:
OPTIONAL: explicit ``str`` is not needed when using ``format``.

Reply:
Thanks, done.